datalad/support/sshconnector.py

```diff
@@ -96,11 +99,14 @@ def open(self):
         connection, if it is not there already.
         """
 
+        # set control options
+        ctrl_options = ["-o", "ControlMaster=auto", "-o", "ControlPersist=yes"] + self.ctrl_options
+        # create ssh control master command
+        cmd = ["ssh"] + ctrl_options + [self.host, "exit"]
+
         # start control master:
-        cmd = "ssh -o ControlMaster=auto -o \"ControlPath=%s\" " \
-              "-o ControlPersist=yes %s exit" % (self.ctrl_path, self.host)
         lgr.debug("Try starting control master by calling:\n%s" % cmd)
-        proc = Popen(cmd, shell=True)
+        proc = Popen(cmd)
```

Comment:
No, sadly I don't remember. There was a reason for this, but may be it's a leftover from previous implementation. If the tests pass, it should be fine without.